src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java

```diff
@@ -5,11 +5,16 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import uk.gov.hmcts.reform.civil.enums.YesOrNo;
+
 @Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
 public class RespondentLiPResponse {
 
+    private YesOrNo partialAdmissionAlreadyPaid;
```

Comment:
Not sure whether these should be just as fields as are to do with partial admission. would not it be better to have a model for lop partial admission and have the fields there?